prefq/server.py

```diff
@@ -47,7 +47,8 @@
 app.config["VIDEO_FOLDER"] = "videos"
 
 feedback_data = {}
-pending_queries = queue.Queue()
+query_queue = queue.Queue()
+query_list = []
```

Comment:
Please give the two variables names that indicate what they are intended to store (something like `queries_not_yet_served` and `queries_pending_response`).

Reply:
done!